content/authoring/kata/read-solution-file.md

```diff
@@ -0,0 +1,23 @@
+---
+kind: recipe
+---
+
+# How to Read a Solution File
+
+Some kata wish to analyze content of the solution file, to verify some requirements related to the code. For example, tests of a code golf kata would probably want to verify the length of submitted solution.
+
+## Location
+
+The submitted solution is written to a file located at path `/workspace/solution.txt`. It can be read by file I/O procedures available in the language of the tests.
```

Comment:
```suggestion
The submitted solution is written to a file located at `/workspace/solution.txt`. It can be read by file I/O procedures available in the language of the tests.
```